store/storetest/user_store.go

```diff
@@ -3357,7 +3357,8 @@ func testUserStoreGetProfilesNotInTeam(t *testing.T, ss store.Store) {
 
 	// Add u2 to team 1
 	store.Must(ss.Team().SaveMember(&model.TeamMember{TeamId: teamId, UserId: u2.Id}, -1))
-	u2.UpdateAt = store.Must(ss.User().UpdateUpdateAt(u2.Id)).(int64)
+	u2.UpdateAt, _ = ss.User().UpdateUpdateAt(u2.Id)
```

Comment:
You still have to "capture" the error here, like `u2.UpdateAt, err = ss.User().UpdateUpdateAt(u2.Id)`, if not, you are checking for a previous error not for the expected one (same for the other calls later).